AdminWebsite/AdminWebsite/Controllers/StaffMemberController.cs

```diff
@@ -0,0 +1,70 @@
+﻿using BookingsApi.Client;
+using BookingsApi.Contract.Requests;
+using BookingsApi.Contract.Responses;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text.Encodings.Web;
+using System.Threading.Tasks;
+
+namespace AdminWebsite.Controllers
+{
+    /// <summary>
+    /// Responsible for retrieving and storing person information
+    /// </summary>
+    [Produces("application/json")]
+    [Route("api/staffmember")]
```

Comment:
I can see that the SuitabilityAnswersController uses suitability-answers for the route, should we follow that naming convention here?

Reply:
I will take your feedback, will do it next time onwards.